chef/cookbooks/barclamp/libraries/barclamp_library.rb

```diff
@@ -150,6 +150,12 @@ def self.unclaimed(node, include_mounted=false)
               %x{lsblk #{d.name.gsub(/!/, "/")} --noheadings --output MOUNTPOINT | grep -q -v ^$}
               next if $?.exitstatus == 0
             end
+            # we want disks that are multipath but not if they are holders
+            # of a multipath
+            if d.multipath?
```

Comment:
The original patch:
```
+            next unless d.fixed || d.multipath?
+            !d.claimed? && !d.multipath_path?
```

Suggested:
```
next if d.claimed? || held_by_multipath?
d.fixed || d.multipath?
```

"fixed" devices and multipath must be included; claimed and paths (held) must be skipped.